src/charts/pie.coffee

```diff
@@ -57,8 +57,7 @@ Ember.Charts.PieComponent = Ember.Charts.ChartComponent.extend(
       value: d.value
       percent: d3.round( 100 * d.value / total )
 
-    key = @get 'sortKey'
-    data.sortBy key
+    _.sortBy data, @get('sortKey')
```

Comment:
I was confused by "doesn't work," I thought you meant there was an error when `data` was undefined based on the PR title.

What error exactly is happening?